src/localize.ts

```diff
@@ -23,7 +23,24 @@ export class Localize {
     const message: string = languagePack[key] || key;
     return this.format(message, args);
   }
-  public async init() {
+  public async init(userFolder: string) {
+    this.options = {
+      locale: "en"
+    };
+
+    try {
+      const pathToLocale = path.resolve(userFolder, "locale.json");
+      if (fs.existsSync(pathToLocale)) {
```

Comment:
if path is not available then we should read the env. variable ? What do you suggest

Reply:
If the file doesn't exist, then VSCode is using the default language (en). I think we should leave it up to the `locale.json` file to have more consistency with VSCode's localizations.